### ISSUE TITLE
GOG Authentication fails

### ISSUE THREAD

**Issue by floreal** [lutris/lutris#2686]
**Describe the bug**

When I try to connect to my gog account through Lutris GUI, nothing happens, no game is displayed. Once done, import dialog won't appear anymore until removing $HOME/.cache/lutris/.gog.auth

Here are the logs I get:

**Expected behavior**

Game list is displayed

**Current behavior**

Nothing happens

**Steps to reproduce**

- Click on import games
- Select GOG tab
- Click on Connect you Account
- Enter gog credential in the pop-up and 2fa code

**Lutris debugging output**

```
2020-03-27 12:06:35,927: /home/floreal/.PlayOnLinux/wine/linux-x86/bin/wine
INFO     2020-03-27 12:06:35,992 [application.do_command_line:225]:Running Lutris 0.5.4
INFO     2020-03-27 12:06:35,993 [startup.check_driver:54]:Using NVIDIA drivers 440.64 for x86_64
INFO     2020-03-27 12:06:35,993 [startup.check_driver:60]:GPU: GeForce GTX 970
INFO     2020-03-27 12:06:35,993 [startup.check_driver:77]:GPU: 10DE:13C2 1458:367A using nvidia drivers
INFO     2020-03-27 12:06:36,033 [startup.check_vulkan:132]:Vulkan is supported
INFO     2020-03-27 12:06:36,034 [dxvk.get_dxvk_versions:21]:Updating DXVK versions
INFO     2020-03-27 12:06:36,257 [dxvk.get_dxvk_versions:21]:Updating D9VK versions
DEBUG    2020-03-27 12:06:36,311 [http.get:65]:GET https://lutris.net/api/runtime
INFO     2020-03-27 12:06:36,311 [lutriswindow.toggle_connection:590]:Connected to lutris.net as Floréal
DEBUG    2020-03-27 12:06:36,321 [http.get:65]:GET https://lutris.net/api/games/library/Floréal
ERROR    2020-03-27 12:06:36,332 [jobs.target:32]:Error while completing task <function sync_from_remote at 0x7f8354b383a0>: 'ascii' codec can't encode character '\xe9' in position 27: ordinal not in range(128)
<class 'UnicodeEncodeError'> 'ascii' codec can't encode character '\xe9' in position 27: ordinal not in range(128)
  File "/usr/lib/python3.8/site-packages/lutris/util/jobs.py", line 30, in target
    result = self.function(*args, **kwargs)
  File "/usr/lib/python3.8/site-packages/lutris/sync.py", line 87, in sync_from_remote
    remote_library = api.get_library()
  File "/usr/lib/python3.8/site-packages/lutris/api.py", line 88, in get_library
    response = request.get()
  File "/usr/lib/python3.8/site-packages/lutris/util/http.py", line 71, in get
    request = urllib.request.urlopen(req, timeout=self.timeout)
  File "/usr/lib/python3.8/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.8/urllib/request.py", line 525, in open
    response = self._open(req, data)
  File "/usr/lib/python3.8/urllib/request.py", line 542, in _open
    result = self._call_chain(self.handle_open, protocol, protocol +
  File "/usr/lib/python3.8/urllib/request.py", line 502, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.8/urllib/request.py", line 1362, in https_open
    return self.do_open(http.client.HTTPSConnection, req,
  File "/usr/lib/python3.8/urllib/request.py", line 1319, in do_open
    h.request(req.get_method(), req.selector, req.data, headers,
  File "/usr/lib/python3.8/http/client.py", line 1230, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/usr/lib/python3.8/http/client.py", line 1241, in _send_request
    self.putrequest(method, url, **skips)
  File "/usr/lib/python3.8/http/client.py", line 1096, in putrequest
    self._output(self._encode_request(request))
  File "/usr/lib/python3.8/http/client.py", line 1176, in _encode_request
    return request.encode('ascii')
DEBUG    2020-03-27 12:06:47,552 [gog.connect:262]:Connecting to GOG
DEBUG    2020-03-27 12:12:55,965 [http.get:65]:GET https://auth.gog.com/token?client_id=46899977096215655&client_secret=9d85c43b1482497dbbce61f6e4aa173a433796eeae2ca8c5f6129f2dc4de46d9&grant_type=authorization_code&code=IbqNZ4Vv_A-waGCg09h1zY9vZY5hUo7pCWxR89-fBX7FnBgVNrcp2haXG22814dunmvfX2-LkSqfLTd1gLZuvm8nbhbiThmKoVv81iz5aKQLhAT-cgCK3ovA4aeAvhZ731WudgGZRm5M6-M3LErtk4iGdzkMgdJFjXtS8pxowgo&redirect_uri=https%3A%2F%2Fembed.gog.com%2Fon_login_success%3Forigin%3Dclient
DEBUG    2020-03-27 12:12:56,144 [gog.load_cookies:111]:Loading cookies from /home/floreal/.cache/lutris/.gog.auth
/usr/lib/python3.8/site-packages/lutris/util/cookies.py:79: UserWarning: http.cookiejar bug!
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/lutris/util/cookies.py", line 31, in _really_load
    domain, domain_specified, path, secure, expires, name, value = line.split(
ValueError: too many values to unpack (expected 7)

  _warn_unhandled_exception()
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/lutris/util/cookies.py", line 31, in _really_load
    domain, domain_specified, path, secure, expires, name, value = line.split(
ValueError: too many values to unpack (expected 7)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/lutris/gui/widgets/services.py", line 137, in on_connect_clicked
    self._connect_button_toggle()
  File "/usr/lib/python3.8/site-packages/lutris/gui/widgets/services.py", line 144, in _connect_button_toggle
    if self.service.is_connected():
  File "/usr/lib/python3.8/site-packages/lutris/services/gog.py", line 257, in is_connected
    return GOG_SERVICE.is_available()
  File "/usr/lib/python3.8/site-packages/lutris/services/gog.py", line 70, in is_available
    user_data = self.get_user_data()
  File "/usr/lib/python3.8/site-packages/lutris/services/gog.py", line 170, in get_user_data
    return self.make_api_request(url)
  File "/usr/lib/python3.8/site-packages/lutris/services/gog.py", line 155, in make_api_request
    request = Request(url, headers=headers, cookies=self.load_cookies())
  File "/usr/lib/python3.8/site-packages/lutris/services/gog.py", line 116, in load_cookies
    cookiejar.load()
  File "/usr/lib/python3.8/http/cookiejar.py", line 1799, in load
    self._really_load(f, filename, ignore_discard, ignore_expires)
  File "/usr/lib/python3.8/site-packages/lutris/util/cookies.py", line 80, in _really_load
    raise OSError(
OSError: invalid Netscape format cookies file '/home/floreal/.cache/lutris/.gog.auth': 'login.gog.com\tFALSE\t/\tTRUE\t1602587239\tgalaxy-login-s\taqlc9tercjpu70g7d8mnb698g0\tNone'
INFO     2020-03-27 12:15:57,948 [application.do_shutdown:450]:Shutting down Lutris
```

**System information**

[lutris-issue-report.json.gz](https://github.com/lutris/lutris/files/4393128/lutris-issue-report.json.gz)

**Comment by connectety** [comment]
This seams like this is very likely a duplicate of #2645, which is caused by an update to the GOG authentication.
The error was fixed with the commit 78a6422 and the fix was improved in f27f61b.
This fix sadly is only available in git (maybe there will be a release soon).
For the time being, **maybe** edit the files manually, by appending `, *_extra` after to `/usr/lib/python3/dist-packages/lutris/util/cookies.py` (or your distros version of this) in line 31. 
Arch Linux and other distros sometimes offer fixes for these kind of issues in their distribution of lutris.
As a Arch Linux user you problably can also use the lutris-git package for fixes like this.

**Comment by tannisroot** [comment]
Fixed in 0.5.5
